Sources/StreamChatUI/Gallery/GalleryVC.swift

```diff
@@ -46,6 +46,9 @@ open class GalleryVC:
         return videos + images
     }
     
+    /// Returns the date formater function used to represent when the user was last seen online
+    open var lastSeenDateFormatter: (Date) -> String? { DateUtils.timeAgo }
```

Comment:
Instead of this, why don't we inject this via `Components`? we do the same injection mechanism for 
- localizationProvider
- channelNamer
- messageLayoutOptionsResolver

Reply:
Components is growing like crazy, I think for small things like this is better not to use it (also in this case we'd need to have one for the gallery and one for the channel header)